setup.json

```
@@ -7,7 +7,7 @@
     ],
     "description": "Package that integrates AiiDAlab with Electronic Laboratory Notebooks.",
     "install_requires": [
-        "aiida-core>=1.1.0,<2.0.0",
+        "aiida-core>=1.1.0,<3.0.0",
```

I believe that this should suffice:
```suggestion
        "aiida-core>=1.1.0,<3",
```